### DISCUSSION TITLE
UMP implementation ready for new IAB TCF requirement

### DISCUSSION THREAD

**Discussion by Telestes** [Poing-Studios/Godot-AdMob-Android-iOS#169] [Q&A]
Hi gumaciel!

Is your current UMP implementation ready for the new IAB TCF requirements published by Google?
[https://blog.google/products/admanager/new-cmp-requirement/](url)

At the moment I'm using V2.1.2 of your Godot-AdMob-Android plugin.

Thank you, Telestes

**Comment by gumaciel** [comment]
Hi @Telestes, I believe we are, we are using the User Messaging Platform version 2.0.

I don't know if we need to do something extra in our implementation, if you know please let us know

**Reply by Momo90k** [reply]
Is CCPA also supported alongside GDPR?
